### PR TITLE
Hotfix/v1.0.1

### LINKED ISSUE #1151: ErrorBoundary is broken and points to uniswap
**Description**
When there's an error, the new version doesn't show the white page anymore, that's great!, however shows something that seems broken too.

On top of this, shows some links to Github and Discord pointing to Uniswap instead of CowSwap.

image

## Technical details
Relevant component: `ErrorBoundary`

## Good to have
Ideally we would do also https://github.com/gnosis/cowswap/issues/339

### PR DESCRIPTION
**Pull request by anxolin**
# Summary

Closes #1151

Light mode:
image

Dark mode:
image

# Out of scope
* Add a header to navigate to the home
* Nicer styles, we should do in `develop` a new PR using a cow Meme

# To Test
I've made the Cow Game page to throw an error so we can see the app crashing.

1. Try to play the game (use the menu to navigate)
2. Observe the error page, make sure there's no traces of Uniswap

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @anxolin , just like an enhancement: could we add a label `app:Cowswap` + `Critical` and something like `crash report` or so in order to track these issues on the board better?

Nevertheless, changes LGTM!

**Comment by alongoni** [comment]
Looks good! 
What about if we include the header in the error page. So users can go back to the swap page or use the menu.
image


Github and Discord link are ok!

**Comment by biocom** [comment]
Looks good. Same thoughts as @alongoni

**Review by biocom — APPROVED** [review]
Approve with a comment. Not sure if you intend to address in this PR.

File: src/custom/components/ErrorBoundary/ErrorBoundaryMod.tsx [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+const FallbackWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  align-items: center;
+  z-index: 1;
+`
+
+const BodyWrapper = styled.div<{ margin?: string }>`
+  padding: 1rem;
```
Comment (biocom) [inline src/custom/components/ErrorBoundary/ErrorBoundaryMod.tsx:20]: Could use px's to match other values.

File: src/custom/components/ErrorBoundary/ErrorBoundaryMod.tsx [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+                  <TYPE.main fontSize={10} color={'text1'}>
+                    {error.stack}
+                  </TYPE.main>
+                </code>
+              </CodeBlockWrapper>
+              <AutoRow>
+                <LinkWrapper>
+                  <ExternalLink
+                    id="create-github-issue-link"
+                    // href={`https://github.com/Uniswap/uniswap-interface/issues/new?assignees=&labels=bug&body=${encodedBody}&title=${encodeURIComponent(
```
Comment (biocom) [inline src/custom/components/ErrorBoundary/ErrorBoundaryMod.tsx:96]: Comment to be removed, or?

**Comment by anxolin** [comment]
@alongoni @biocom , yep, agree on that.

Maybe do that in another branch that ends up merging in develop.

My concern with your suggestion is that this is a `hotfix` that will go to production, and adding the normal header might introduce some issue. It can happen that the original error that triggered in the original header (maybe some hook logic)

Once this is merged in develop, we can easily iterate

File: src/custom/components/ErrorBoundary/ErrorBoundaryMod.tsx (hunk 2 above)
Comment (W3stside) [reply src/custom/components/ErrorBoundary/ErrorBoundaryMod.tsx:96]: its a mod.

File: src/custom/pages/CowGame/index.tsx [hunk 3]
```diff
@@ -30,6 +30,10 @@ const Wrapper = styled(Page)`
 `
 
 export default function CowGamePage() {
+  if (0 === 0) {
```
Comment (W3stside) [inline src/custom/pages/CowGame/index.tsx:33]: ????

**Comment by elena-zh** [comment]
2 points related to the Mobile view: 
1. Is that OK that error log container is to wide?
2. It would be nice to increase links visibility, as they are displayed above the cow pic and almost not visible
image
image

File: src/custom/pages/CowGame/index.tsx (hunk 3 above)
Comment (anxolin) [reply src/custom/pages/CowGame/index.tsx:33]: will be reverted, its too force an error. Briefly commented in the test description

**Comment by anxolin** [comment]
@elena-zh good points. For this PR we will just deploy the hotfix of the issue of the repo and not being able to read. 
We are in the middle of other things, and we need to deploy it to production. This issues you mentioned are already present in production. `Nicer styles` was removed from the scope

**Review by W3stside — APPROVED** [review]

**Review by elena-zh — APPROVED** [review]
LGTM!
I have created a separate issue for the nicer styles for this page #1255

File: src/custom/components/ErrorBoundary/ErrorBoundaryMod.tsx (hunk 1 above)
Comment (anxolin) [reply src/custom/components/ErrorBoundary/ErrorBoundaryMod.tsx:20]: It's a mod, i didn't touch this part. We can probably redo this component completly. For now just hotfix

**Pull request merged**